### DISCUSSION TITLE
Where are the CSLA 8.0.0-R24031302 nuget packages that the ProjectTracker is referencing?

### DISCUSSION THREAD

**Discussion by Tony20221** [MarimerLLC/csla#3739] [Questions]
I am getting errors during compilation of ProjectTracker because CSLA 8.0.0-R24031302 packages don't exist. They are not on my machine and not on Nuget.org.

**Comment by rockfordlhotka** [comment]
I forgot to push them 🫢

**Comment by sshushliapin** [comment]
I'm having a similar error with the BlazorExamples project in the main branch:

```
Error (active)	NU1102	Unable to find package Csla with version (>= 9.0.0-R24051102)
```

**Comment by rockfordlhotka** [comment]
I think the real solution is this: https://github.com/MarimerLLC/csla/issues/2543

Right now we're often working on new features in CSLA, and tend to use a local "NuGet feed" from the local hard drive. What happens is that the sample code gets pushed to GitHub, but it isn't realistic to flood NuGet with multiple prerelease packages per week (or sometimes per day).

If we have a NuGet package feed from _GitHub_ for the purposes of testing and prereleases, that'd be realistic and manageable, and something that could perhaps become part of the CI build process.
